doc/client_lua_api.md

```diff
@@ -648,10 +648,10 @@ Call these functions only at load time!
       semi-frequent intervals as well as on server shutdown.
 * `minetest.register_on_connect(func())`
     * Called at the end of client connection (when player is loaded onto map)
-* `minetest.register_on_receiving_chat_message(func(name, message))`
+* `minetest.register_on_receiving_chat_messages(func(message))`
```

Comment:
Well then, @DS-Minetest please change `s_client.cpp` to call `registered_on_receiving_chat_message` instead of the version with `s`, for consistency with the other callback functions.